big-red-shuttle/StopsViewController.swift

```diff
@@ -61,7 +64,33 @@ class StopsViewController: UIViewController, GMSMapViewDelegate {
             marker.map = mapView
         }
     }
+    
+    
+    //drawing
+    func initPolyline() {
+        let stops = getStops()
+        let stopsA = Array(stops[1..<9]) //can only have a max of 8 waypoints (i.e. stops that don't include start and end)
+        
+        self.polyline.getPolyline(waypoints: stopsA, origin:stops[0], end:stops[9])
+        self.polyline2.getPolyline(waypoints: [], origin: stops[9], end: stops[0])
+        self.drawRoute()
+    }
 
+    func drawRoute() {
+        let path1 = GMSMutablePath(fromEncodedPath: polyline.overviewPolyline)
+        let path2 = GMSMutablePath(fromEncodedPath: polyline2.overviewPolyline)
+        let routePolyline = GMSPolyline(path: path1)
+        let routePolyline2 = GMSPolyline(path: path2)
+        
+        routePolyline.strokeColor = UIColor(red: 206/255, green: 73/255, blue: 55/255, alpha: 1.0)
+        routePolyline2.strokeColor = UIColor(red: 206/255, green: 73/255, blue: 55/255, alpha: 1.0)
```

Comment:
These colors will be added into the UIColor+Shared.swift file when Monica merges her stuff